core/frame/data_frame.go

```diff
@@ -2,15 +2,10 @@ package frame
 
 import (
 	"fmt"
-	"sync"
 
 	"github.com/yomorun/y3"
 )
 
-// dataFramePool provides a syncPool for dataFrame.
-// Because that dataFrame is the most frequently allocated object.
-var dataFramePool sync.Pool
```

Comment:
The issue still lies with the dataFrame pool. I discovered during testing (example-6) that the dataTags were being lost. After careful consideration, I delete this pool for now as it was causing too many issues. @venjiang